packages/babel-plugin-transform-typescript/src/enum.ts

```diff
@@ -193,12 +193,12 @@ export function translateEnumValues(
 
 // Based on the TypeScript repository's `evalConstant` in `checker.ts`.
 function evaluate(
-  expr,
+  expr: t.Node,
   seen: PreviousEnumMembers,
 ): number | string | typeof undefined {
   return evalConstant(expr);
 
-  function evalConstant(expr): number | typeof undefined {
+  function evalConstant(expr: t.Node): number | typeof undefined {
```

Comment:
a same thing

Reply:
Yeah. I am good to leave it as-is. I can open a new PR replacing `typeof undefined` to `undefined` later.